site/src/sphinx/server-grpc.rst

```diff
@@ -200,6 +202,29 @@ entire service implementation in ``RequestContext.current().blockingTaskExecutor
         }
     }
 
+.. code-block:: java
+
+    import com.linecorp.armeria.common.RequestContext;
+    import com.linecorp.armeria.server.ServiceRequestContext;
+    import com.linecorp.armeria.server.grpc.GrpcServiceBuilder;
+
+    public class MyHelloService extends HelloServiceGrpc.HelloServiceImplBase {
+        @Override
+        public void hello(HelloRequest req, StreamObserver<HelloReply> responseObserver) {
+            Thread.sleep(10000);
+            HelloReply reply = HelloReply.newBuilder()
+                                         .setMessage("Hello, " + req.getName() + '!')
+                                         .build();
+            responseObserver.onNext(reply);
+            responseObserver.onCompleted();
+        }
+
+        public static void configureService(GrpcServiceBuilder builder) {
+            // All service methods will be run within the blocking executor.
+            builder.useBlockingExecutor(true);
```

Comment:
Would you mind updating this example like this:

```java
public class MyHelloService ... {
   ...
}

ServerBuilder sb = new ServerBuilder();
sb.service(new GrpcServiceBuilder().addService(...)
                                   .useBlockingExecutor(true)
                                   .build());
```

.. so that we do not give a user an impression that `configureService` will be automatically invoked?

Reply:
Sure - wasn't sure if the doc snippets are supposed to compile by itself or not

Reply:
It's OK even if it doesn't compile in my opinion. We could add a fully working example to `examples/` and add a link in `.rst`, though.